### DISCUSSION TITLE
Reporting status: 'Reported only' after ‘Exploring data sources’?

### DISCUSSION THREAD

**Discussion by karihj** [open-sdg/open-sdg#1599] [Q&A] [answered]
On out reporting status site: https://hagstovan.github.io/sdg-site/en/reporting-status/ the ‘Reported online’ is showing after ‘Exploring data sources’ in the given line for each goal. 
Is there away to switch this around so ‘Reported online’ will show first on the line and then ‘Exploring data sources’?

Thank you!

**Comment by brockfanning** [comment, marked as answer]
@karihj Let's see if the order in the site configuration might fix it. Can you try changing the order of the items in the `reporting_status` `status_types` setting? That would be here in your site config: https://github.com/hagstovan/sdg-site/blob/develop/_data/site_config.yml#L233-L242

**Reply by karihj** [reply]
@brockfanning That did the trick! Thank you!
